### PR TITLE
Jhc/setup.sh

### PR DESCRIPTION
**Pull request by jeremyhcho**
edit setup file to include homebrew and golang

### CODE REVIEW

File: scripts/setup.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+which -s brew
+if [[ $? != 0 ]] ; then
+    echo "Homebrew not installed, installing..."
+    ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+else
+    echo "Homebrew already installed, checking for update..."
+    brew update
+fi
+
+# Check if Go exists, install it if not.
```
Comment (paulcho8) [inline scripts/setup.sh:11]: I don't think you're actually checking to see if go exists? `$?` doesn't read anything.

Does `go version` suffice? This should error out if go isn't installed.

Comment (paulcho8) [inline scripts/setup.sh:11]: I don't think you're actually checking to see if go exists? `$?` doesn't read anything.

Does `go version` suffice? This should error out if go isn't installed.

**Review by paulcho8 — CHANGES_REQUESTED** [review]

File: scripts/setup.sh (hunk 1 above)
Comment (jeremyhcho) [reply scripts/setup.sh:11]: oh oops good catch

Comment (jeremyhcho) [reply scripts/setup.sh:11]: u sexy man

Comment (jeremyhcho) [reply scripts/setup.sh:11]: done

**Review by paulcho8 — APPROVED** [review]
LGTM

**Review by mkshin1 — APPROVED** [review]

**Pull request merged**